frontend/common/src/components/skills/MissingSkills/MissingSkills.tsx

```diff
@@ -11,7 +11,7 @@ export interface MissingSkillsProps {
   addedSkills?: Skill[];
 }
 
-const getMissingSkills = (required: Skill[], added?: Skill[]) => {
+export const getMissingSkills = (required: Skill[], added?: Skill[]) => {
```

Comment:
Would it be a good idea to move this to `frontend/common/src/helpers` if we are going to be using it in different places?

Reply:
Fixed by [Move reusable functions into helper folders](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3863/commits/3aae2129d3f34a9a21bcf9b33d6a421e3e73480d)